app/assets/javascripts/dragdrop.js

```diff
@@ -51,13 +51,19 @@ jQuery(document).ready(function() {
                 'nid':$D.nid
             },
             start: function(e) {
+                $('#imagebar .inline_image_drop').show();
+                $('#create_progress').show();
+                $('#create_uploading').show();
                 elem = $($D.selected).closest('div.comment-form-wrapper').eq(0);
                 elem.find('#create_progress').eq(0).show();
                 elem.find('#create_uploading').eq(0).show();
                 elem.find('#create_prompt').eq(0).hide();
                 elem.find('.dropzone').eq(0).removeClass('hover');
             },
             done: function (e, data) {
```

Comment:
This PR has a codeclimate issue. The `done` function is of 27 lines whereas the limit is 25. How can it be optimized further?

Reply:
Remove blank line 73

Reply:
Declare multiple variables in one line.